webapp/src/main/webapp/WEB-INF/csrfguard/Owasp.CsrfGuard.properties

```diff
@@ -0,0 +1,495 @@
+######################################################################################
```

Comment:
What do you think of simplifying this file name to something like `csrfguard.properties`?